docs/support/index.rst

```diff
@@ -6,26 +6,22 @@ General support matrix
 
 
 
-  =====================   ==========  =============   ============ ==============  ============  ============  ===============  =========================  ==============  ==============
```

Comment:
These changes should be already tracked under https://github.com/napalm-automation/napalm/pull/485